.gitignore

```diff
@@ -149,3 +149,4 @@ cache
 .cache
 .idea/
 wandb/
+**/resources
```

Comment:
not the biggest fan of adding this broad a filter to `.gitignore`, e.g. what if some data plugin includes `data/<plugin>/resources/download_data.py` that would be automatically excluded. If it's okay with you, let's remove (or make it more specific).
```suggestion
```

Reply:
Resolved in the latest commit ([f631e5b](https://github.com/brain-score/language/pull/88/commits/f631e5b86364d9ef4d58539baa52ae6f03664bcf))